src/modules/jcms_notifications/drush/jcms_notifications.drush.inc

```diff
@@ -98,11 +113,59 @@ function drush_jcms_notifications_article_import_all() {
 }
 
 /**
+ * Callback function drush_jcms_notifications_article_metrics_import_all().
+ */
+function drush_jcms_notifications_article_metrics_import_all() {
+  $metrics_service = \Drupal::service('jcms_article.fetch_article_metrics');
+  drush_print(dt('Fetching article metrics. This may take a few minutes.'));
+  $limit = drush_get_option('limit') ? (int) drush_get_option('limit') : NULL;
+
+  $query = \Drupal::entityQuery('node')
+    ->condition('status', \Drupal\node\NodeInterface::PUBLISHED)
+    ->condition('changed', \Drupal::time()->getRequestTime(), '<')
+    ->condition('type', 'article');
+  if (!empty($limit)) {
+    $query->range(0, $limit);
+  }
+  if (drush_get_option('skip-updates')) {
+    $query->condition('field_page_views.value', 0);
+  }
+  $nids = $query->execute();
+  /* @var Node[] $nodes */
+  $nodes = Node::loadMultiple($nids);
+  drush_print(dt('Received !count article metrics to process.', ['!count' => count($nids)]));
+  if ($nodes) {
+    $time_start = microtime(TRUE);
+    $num = 0;
+    foreach ($nodes as $nid => $node) {
+      $articleMetrics = $metrics_service->getArticleMetrics($node->label());
+      if ((int) $node->get('field_page_views')->getString() != $articleMetrics->getPageViews()) {
+        $node->set('field_page_views', $articleMetrics->getPageViews());
+        $node->save();
+      }
+      drush_print(dt('Processed article metrics for !article_id (!num of !count)', [
+        '!article_id' => $node->label(),
+        '!num' => ++$num,
+        '!count' => count($nids),
+      ]));
+    }
+    $time_end = microtime(TRUE);
+    $time = round($time_end - $time_start, 0);
+    drush_print(dt('Processed !count article metrics in !minutes minutes !seconds seconds.', [
+      '!count' => count($nids),
+      '!minutes' => floor($time / 60),
+      '!seconds' => round($time % 60),
+    ]));
+  }
+}
+
+/**
  * Callback function drush_jcms_notifications_article_import().
  */
 function drush_jcms_notifications_article_import($lrp = FALSE) {
```

Comment:
the name of this drush command is outdated, but let's not rename it now as it involves the formula. We'll do that incrementally